### PR TITLE
Use fieldMode for creators in OpenURL

### PR DESCRIPTION
**Pull request by AbeJellinek**
There is no `isInstitution` field, so OpenURL search was producing invalid data for items with institutional authors.

### CODE REVIEW

**Comment by AbeJellinek** [comment]
https://github.com/zotero/translators/pull/2404#issuecomment-853684689

File: chrome/content/zotero/xpcom/openurl.js [hunk 1]
```diff
@@ -407,7 +407,7 @@ Zotero.OpenURL = new function() {
 				
 				item.creators.push(_cloneIfNecessary(Zotero.Utilities.cleanAuthor(value, type, value.indexOf(",") !== -1), item));
 			} else if(key == "rft.aucorp") {
-				complexAu.push(_cloneIfNecessary({lastName:value, isInstitution:true}, item));
+				complexAu.push(_cloneIfNecessary({lastName:value, fieldMode:1}, item));
```
Comment (dstillman) [inline chrome/content/zotero/xpcom/openurl.js:410]: This one might be correct, though, if it's meant to be returning internal JSON (which is what "the format returned by item.fromArray()" basically means).

File: chrome/content/zotero/xpcom/openurl.js [hunk 2]
```diff
@@ -210,7 +210,7 @@ Zotero.OpenURL = new function() {
 				_mapTag(firstCreator.firstName, "invfirst");
 				_mapTag(firstCreator.lastName, "invlast");
 			} else {
-				if(firstCreator.isInstitution) {
+				if(firstCreator.fieldMode) {
```
[the remark this answers is not in the document]
Comment (AbeJellinek) [reply chrome/content/zotero/xpcom/openurl.js:213]: Interesting, OK. Should the predicate be `firstCreator.name` and the line below be `_mapTag(firstCreator.name, "aucorp");`?

[the remark this answers is not in the document]
Comment (dstillman) [reply chrome/content/zotero/xpcom/openurl.js:213]: Yes, that's right.

[the remark this answers is not in the document]
Comment (AbeJellinek) [reply chrome/content/zotero/xpcom/openurl.js]: OK, fixed.

**Pull request merged**